openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -509,6 +509,64 @@ def _default_settings(cls):
             output_settings=MultiStateOutputSettings(),
         )
 
+    @classmethod
+    def _adaptive_settings(
+            cls,
+            stateA: ChemicalSystem,
+            stateB: ChemicalSystem,
+            mapping: gufe.LigandAtomMapping | list[gufe.LigandAtomMapping]
+    ) -> RelativeHybridTopologyProtocolSettings:
+        """
+        Get the recommended OpenFE settings for this protocol based on the input states involved in the
+        transformation.
+
+        These are intended as a suitable starting point for creating an instance of this protocol, which can be further
+        customized before performing a Protocol.
+
+        Notes
+        -----
+            The fast settings used are still being refined and tested and will only be applied
+            to net neutral transformations.
+        """
+        # get an unfrozen copy of the default settings that we can edit
+        protocol_settings = cls.default_settings().unfrozen_copy()
```

Comment:
Do you know why this is necessary? The return from default_settings shouldn't be frozen 👀